### DISCUSSION TITLE
Can I host a Fresh site everywhere?

### DISCUSSION THREAD

**Discussion by rempas** [freshframework/fresh#2724] [Q&A] [answered]
Hello everyone! I'm trying to learn Fresh and become a freelancer and one of the things I don't understand (I'm also a beginner when it comes to webdev) is how are Fresh sites hosted. Can I host everyone, or I'm forced to use [deno deploy](https://deno.com/deploy)?

I also don't understand terms like "serverless" (how can something not have a server) and "runtime" (not like the JS runtime, but the server runtime or whatever) but, I'm going to research those terms in the meantime until the question is answered.

**Comment by marvinhagemeister** [comment, marked as answer]
Yes, a Fresh site can be hosted anywhere where Deno can run. We have a page in our docs on how to host it with Docker: https://fresh.deno.dev/docs/concepts/deployment#-docker

**Reply by rempas** [reply]
Thank you! That will allow me to search for what I want! Have a beutiful day!
